.gitignore

```diff
@@ -8,6 +8,7 @@ npm-debug.log
 *.sublime-*
 coverage
 .DS_Store
+.mason
```

Comment:
I'm not sure what actually creates these files, I assume it's the test suite. I added it to the `.gitignore` because they were on my disk, shouldn't be checked-in and already excluded by `.npmignore` from being published.